### ISSUE TITLE
Differences in rsync packages

### ISSUE THREAD

**Issue by Olzhas011** [ansible-lockdown/RHEL9-CIS-Audit#50]
I am using branch _benchmark_v1.0.0_ RHEL9-CIS-Audit and _main_ branch RHEL9-CIS . Why in cis-ansible package name **rsync-daemon** in cis-goss package name **rsync** ?

https://github.com/ansible-lockdown/RHEL9-CIS/blob/2debbf573ebd95a7ef3d33d9937fdd76bc3266b6/tasks/section_2/cis_2.2.x.yml#L326

https://github.com/ansible-lockdown/RHEL9-CIS-Audit/blob/381057d3bcae896f038ac5c4990f733e92a81396/section_2/cis_2.2/cis_2.2.18.yml#L5

**Comment by gocyclones** [comment]
I agree that this is broken and causes a false assertion for rsync 2.2.18.  The actual benchmark audit should be for the rsync-daemon package and not rsync.  Having the rsync package installed is not in violation of the benchmark.  
The remediation is for rsync-daemon: https://github.com/ansible-lockdown/RHEL9-CIS/blob/ef2b7dca5d8dfdb1bcb7d23fc6393ec24407f71e/tasks/section_2/cis_2.2.x.yml#L326

**Comment by uk-bolly** [comment]
Closing as merged with correct benchmark branch
